cmd/influx/write.go

```diff
@@ -15,20 +15,27 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	inputFormatCsv          = "csv"
+	inputFormatLineProtocol = "lp"
+)
+
 var writeFlags struct {
 	OrgID     string
```

Comment:
can you replace OrgID and Org with the org type that does this:

https://github.com/influxdata/influxdb/blob/master/cmd/influx/main.go#L360

you can see it being used here:

setting the flags: https://github.com/influxdata/influxdb/blob/master/cmd/influx/pkg.go#L100
validating org id: https://github.com/influxdata/influxdb/blob/master/cmd/influx/pkg.go#L108
getting org id from type: https://github.com/influxdata/influxdb/blob/master/cmd/influx/pkg.go#L122

side note: just found redundant code I wrote in that cmd 😬

Reply:
:) ok